.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,26 @@
+name: Generate Release Notes on Tag
+
+run-name: Generate Release Notes for Tag ${{ github.ref_name }}
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release_notes:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Publish Release Notes
+      id: publish_release
+      uses: release-drafter/release-drafter@v6
+      with:
+        version: ${{ github.ref_name }}
+        tag: ${{ github.ref_name }}
+        name: ${{ github.ref_name }}
+        latest: true
```

Comment:
Looks like `latest` is `true` by default - https://github.com/release-drafter/release-drafter?tab=readme-ov-file#action-inputs